app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,558 @@
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.History,
+                        tint = MaterialTheme.colorScheme.onSurface,
+                        contentDescription = null,
+                    )
+                    Spacer(
+                        modifier = Modifier.width(dimensionResource(R.dimen.search_bar_suggestion_content_padding)),
+                    )
+                    Text(
+                        text = suggestion,
+                        style = MaterialTheme.typography.bodySmall,
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun HomeScreenChipGroup(
+    categories: List<ContentType>,
+    onChipClick: (ContentType) -> Unit,
+    modifier: Modifier = Modifier,
+    initialCategory: ContentType = ContentType.MOVIE,
+) {
+    var selectedChip by rememberSaveable {
+        mutableStateOf(initialCategory)
+    }
+    LazyRow(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier,
+    ) {
+        items(categories) { category ->
+            FilterChip(
+                modifier = Modifier
+                    .padding(
+                        start =
+                        if (category == initialCategory || category == categories.last()) {
+                            dimensionResource(
+                                R.dimen.home_screen_horizontal_padding,
+                            )
+                        } else {
+                            dimensionResource(R.dimen.filter_chip_group_content_padding)
+                        },
+                    )
+                    .height(dimensionResource(R.dimen.filter_chip_height))
+                    .width(dimensionResource(R.dimen.filter_chip_width)),
+                selected = selectedChip == category,
+                shape = RoundedCornerShape(dimensionResource(R.dimen.filter_chip_shape_size)),
+                border = null,
+                colors = FilterChipDefaults.filterChipColors(
+                    selectedContainerColor = MaterialTheme.colorScheme.secondary,
+                    selectedLabelColor = MaterialTheme.colorScheme.secondary,
+                    disabledLabelColor = MaterialTheme.colorScheme.onSurface,
+                    disabledSelectedContainerColor = MaterialTheme.colorScheme.onBackground,
+                ),
+                onClick = {
+                    selectedChip = category
+                    onChipClick(selectedChip)
+                },
+                label = {
+                    Text(
+                        text = createChipLabel(category),
+                        style = if (category == selectedChip) {
+                            MaterialTheme.typography.labelSmall
+                        } else {
+                            MaterialTheme.typography.labelSmall.copy(
+                                color = grey_50,
```

Comment:
Pass the colors from the themes color scheme

Reply:
done